samples/matter/light_bulb/sysbuild/multiprotocol_rpmsg/prj.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+## Disable serial and UART interface.
```

Comment:
Would it be possible to inherit this configuration from the Kconfig.multiprotocol_rpmsg.defaults, similarly, as we have it for the child images?

Reply:
This is what I suggested in TODO, but I guess we will need more integration bits for Matter samples to work with OTA and factory data, so this is just an initial effort to make samples working with sysbuild. But this is an important topic, @nordicjm is it possible to inherit domain/image configuration from NCS modules, like connectedhomeip as mentioned?

Reply:
> Would it be possible to inherit this configuration from the Kconfig.multiprotocol_rpmsg.defaults, similarly, as we have it for the child images?

If you mean the old child/parent way of doing things with inheritance, no, not supported, will not be supported

Reply:
So is there a different way to reuse the same multiprotocol configuration for selected samples, e.g. Matter samples? Or do we need to maintain copies of the same .conf file all over the place?

Reply:
Needs copies of the .conf